containers/src/Kokkos_Vector.hpp

```diff
@@ -147,6 +147,72 @@ class vector : public DualView<Scalar*, LayoutLeft, Arg1Type> {
 
   void clear() { _size = 0; }
 
+  void insert(const_iterator it, const value_type& val) { insert(it, 1, val); }
+
+  void insert(const_iterator it, size_type count, const value_type& val) {
```

Comment:
Any reason not to return an iterator to the first inserted element on the host?

Reply:
Other than me not reading the documentation for vector::insert correct ... no. I will change that.